docs/staking-modules/csm/contracts/CSAccounting.md

```diff
@@ -0,0 +1,1107 @@
+# CSAccounting
+
+- [Source code](https://github.com/lidofinance/community-staking-module/blob/fa7ba8a0bab685fc924aa1b135b8d59f4c6de497/src/CSAccounting.sol)
+- [Deployed contract](https://etherscan.io/address/0x4d72BFF1BeaC69925F8Bd12526a39BAAb069e5Da)
+
+CSAccounting.sol is the supplementary contract responsible for the management of bond, rewards, and penalties. It stores bond tokens in the form of stETH shares, provides information about the bond required, and provides interfaces for the penalties.
+
+## Upgradability
+
+The contract uses [OssifiableProxy](../../../contracts/ossifiable-proxy.md) for upgradability.
```

Comment:
I would avoid using relative paths due to some issues with trailing slashes when navigating back and forth through docausaurus content :eyes: